lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -220,7 +146,7 @@ def render_to_fragment(self, request, program_uuid, **kwargs):  # lint-amnesty,
             'industry_pathways': industry_pathways,
             'credit_pathways': credit_pathways,
             'program_discussions_enabled': program_discussions_is_enabled(),
```

Comment:
In this PR or in the new PR -- is this possible to refactor and remove `program_discussions_enabled`? This is similar to `discussion_fragment.enabled`.